raiden-ts/src/raiden.ts

```diff
@@ -500,7 +532,11 @@ export class Raiden {
         }),
       )
       .toPromise();
-    this.store.dispatch(channelOpen({ settleTimeout }, { tokenNetwork, partner }));
+
+    this.deps.config$.subscribe(({ settleTimeout }: RaidenConfig) =>
```

Comment:
I feel like this is wrong as this is going to dispatch a `channelOpen` every time the config changes?

Reply:
Yes, it will.
I think the best place to handle the default `settleTimeout` is in the `channelOpenEpic`, so the request (`channelOpen`) action is dispatched with a possibly undefined `settleTimeout` value, to signal it was called without specifying one, and on that epic you get the config and replace the default value before sending the transaction.